sql/postgresql/PostgreSQLParser.g4

```diff
@@ -2758,22 +2683,15 @@ altercollationstmt
    ;
 
 altersystemstmt
-   : ALTER SYSTEM_P SET generic_set
-   | ALTER SYSTEM_P RESET generic_reset
+   : ALTER SYSTEM_P (SET | RESET) generic_set
    ;
 
 createdomainstmt
    : CREATE DOMAIN_P any_name opt_as typename colquallist
    ;
 
 alterdomainstmt
-   : ALTER DOMAIN_P any_name alter_column_default
-   | ALTER DOMAIN_P any_name DROP NOT NULL_P
-   | ALTER DOMAIN_P any_name SET NOT NULL_P
-   | ALTER DOMAIN_P any_name ADD_P tableconstraint
-   | ALTER DOMAIN_P any_name DROP CONSTRAINT name opt_drop_behavior
-   | ALTER DOMAIN_P any_name DROP CONSTRAINT IF_P EXISTS name opt_drop_behavior
-   | ALTER DOMAIN_P any_name VALIDATE CONSTRAINT name
+   : ALTER DOMAIN_P any_name (alter_column_default | DROP NOT NULL_P | SET NOT NULL_P | ADD_P tableconstraint | DROP CONSTRAINT (IF_P EXISTS)? name opt_drop_behavior | VALIDATE CONSTRAINT name)
```

Comment:
```suggestion
   : ALTER DOMAIN_P any_name
       ( alter_column_default
       | (DROP | SET) NOT NULL_P
       | ADD_P tableconstraint
       | DROP CONSTRAINT (IF_P EXISTS)? name opt_drop_behavior
       | VALIDATE CONSTRAINT name
       )
```

Reply:
see comment above - i do not merge (yet) all possibly alternatives as it do not affect (significantly) parsing of typical SQL script.